tests/Fixer/Phpdoc/PhpdocOrderFixerTest.php

```diff
@@ -139,6 +139,16 @@ public function testNoAnnotations(array $config): void
         $this->doTest($expected);
     }
 
+    /**
```

Comment:
Are all method movements forgotten in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8302 or can they be reverted?

Reply:
I have removed this PR since then and improved it a lot.